perfmetrics/scripts/load_tests/python/load_generator/constants.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Please add a newline after the license text, please make this change in all the files.

Reply:
the linter removes the blank line between license and module description.

Reply:
Okay got it, checked a couple of standard modules, found no newlines there.